docs/details/api.md

```diff
@@ -1,5 +1,13 @@
 # API
 
+## Interactive API documentation 
+To integrate your applications or CI/CD, you can use our online interactive documentation located at 
+
+https://percona-pmm.readme.io/reference/introduction
+
+## Swagger API documentation
```

Comment:
If the docs at readme.io are up-to-date (API methods, I mean), do we need referencing to Swagger at all?

Reply:
As far as I understood from our last meeting, they're not up to date. Roma should confirm.

Reply:
@nastena1606, Swagger is still suitable for reference as Local API doc + backup if readme.io is not accurate or not up to date.

@alexfiru, if it's not up to date - it's still a valuable part of the information for users. There are a lot of usage examples.